dev-db/sqlcl-bin/sqlcl-bin-21.2.0.174.2245.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-bin}"
+MY_P="${MY_PN}-${PV}"
+
+inherit java-pkg-2
+
+DESCRIPTION="Oracle SQLcl is the new SQL*Plus"
+HOMEPAGE="https://www.oracle.com/database/technologies/appdev/sqlcl.html"
+SRC_URI="https://download.oracle.com/otn_software/java/sqldeveloper/${MY_P}.zip"
+RESTRICT="mirror"
```

Comment:
Nice that fetch was dropped, but is only mirror right? No bindist is a bit unusual.

Not that I can make much sense out of this license, but looking into it still make sound it shouldn't be allowed.